autoload/ale/handlers.vim

```diff
@@ -76,6 +75,35 @@ function! ale#handlers#HandleGCCFormat(buffer, lines) abort
     return l:output
 endfunction
 
+function! ale#handlers#HandleCppCheckFormat(buffer, lines)
```

Comment:
Could you add a vader unit test for this function? The other generic handler functions have tests for them now.